x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/edit_policy.helpers.tsx

```diff
@@ -34,9 +54,11 @@ export interface EditPolicyTestBed extends TestBed<TestSubjects> {
 export const setup = async (): Promise<EditPolicyTestBed> => {
   const testBed = await initTestBed();
 
-  const setWaitForSnapshotPolicy = (snapshotPolicyName: string) => {
-    const { component, form } = testBed;
-    form.setInputValue('waitForSnapshotField', snapshotPolicyName, true);
+  const setWaitForSnapshotPolicy = async (snapshotPolicyName: string) => {
+    const { component } = testBed;
+    act(() => {
+      testBed.find('snapshotPolicyCombobox').prop('onChange')!([{ label: snapshotPolicyName }]);
```

Comment:
Suggestion: you should be able to change this line to:

`testBed.find('snapshotPolicyCombobox').simulate('change', [{ label: snapshotPolicyName }])`

Reply:
updated, thank you :)